x-pack/plugins/lens/public/editor_frame_service/editor_frame/workspace_panel/workspace_panel_wrapper.tsx

```diff
@@ -199,10 +237,30 @@ export function WorkspacePanelWrapper({
         className={classNames('lnsWorkspacePanelWrapper stretch-for-sharing', {
           'lnsWorkspacePanelWrapper--fullscreen': isFullscreen,
         })}
+        css={{ height: '100%' }}
```

Comment:
Should we use Emotion here as well?

Reply:
the `css` prop is provided by Emotion, so we're already using it